src/ApiFacade.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TransformationPatternScd;
+
+use Keboola\Component\UserException;
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\TransformationPatternScd\Configuration\GenerateDefinition;
+use Keboola\TransformationPatternScd\Mapping\MappingManager;
+
+class ApiFacade
```

Comment:
Tvoj novy kod som presunul tu a niektore fragmenty som potreboval aj inde, takze je to trocha upravene.